### DISCUSSION TITLE
Help me upvote Babybuddy on TTECK

### DISCUSSION THREAD

**Discussion by bachjessen** [babybuddy/babybuddy#904] [Support]
[https://github.com/community-scripts/ProxmoxVE/discussions/217](https://github.com/community-scripts/ProxmoxVE/discussions/217)

**Comment by MrApplejuice** [comment]
Critical question from my side: What is this? (not meant dismissively)

Woundn't creating a dedicated [terraform backend like the google cloud stuff](https://github.com/babybuddy/babybuddy/tree/master/terraform/gcp-cloud-run) be more goal oriented?

**Comment by bachjessen** [comment]
I totally understand. I just moved from a Docker Compose setup to Proxmox today and installed Home Assistant using TTECK's scripts. It makes it super easy to install directly instead of having it inside a Docker container. So it's just an addition to help more people discover BabyBuddy.

[https://community-scripts.github.io/ProxmoxVE/scripts?id=haos-vm](https://community-scripts.github.io/ProxmoxVE/scripts?id=haos-vm)

**Reply by MrApplejuice** [reply]
Oh... I see, and this is a request to have someone create a TTECK-script then, right? Do you want to remove the home assistant dependency?

Anyway. after reading, I now understand that this is maybe not as "spammy" but just a suggestion to raise awareness in another open source community for integration. Still, my personal hunch was that this sounded a little bit "on the edge" regarding "spam level", when I read this first. Just some feedback from my side.
